### ISSUE TITLE
Don't show answer until TTS is completed when using TTS and "Automatic display answer"

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#3445]
Originally reported on Google Code with ID 2549

```
When you use TTS and have got two different voices,for "question" and "answer" card,you
must set predetermine strict determined time for display length for question and answer,whereby
often you must wait after that TTS stop talking,or card change before TTS stop talking.

Cards schould be automaticly change,reverse immidialtely when TTS say what have to
say.

Sorry for my EN
```

Reported by `conderes` on 2015-03-07 12:41:42

**Comment by rlpowell** [comment]
I have TTS and automatically show answer set, and it is routinely the case that the answer is shown before the question is done, let alone giving me time to decide what the answer should be.  This is crazy making.  :)  What I want is for the "time to show answer" to be applied *after* the TTS finishes.

I might take a look at the code, haven't yet.

**Comment by mikehardy** [comment]
Fixed in #6212
